packages/lib/src/project/SimpleProject.js

```diff
@@ -58,7 +58,7 @@ export default class SimpleProject  {
     delete this.implementations[contractName]
   }
 
-  registerImplementation(contractName, { address, bytecodeHash }) {
+  async registerImplementation(contractName, { address, bytecodeHash }) {
```

Comment:
is it `async`?

Reply:
It's just to be consistent with the equivalent method in AppProject